manifest.json

```
@@ -9,32 +9,44 @@
                     "dst": ".github"
                 },
                 {
-                    "src": "src/${{ language }}/common",
+                    "src": "src/${{ repoType }}",
                     "dst": ".github"
                 },
                 {
-                    "src": "src/${{ language }}/${{ repoType }}",
+                    "src": "src/${{ language }}-common",
+                    "dst": ".github"
+                },
+                {
+                    "src": "src/${{ language }}-${{ repoType }}",
                     "dst": ".github"
                 }
             ],
             "variables": [
                 {
                     "name": "language",
                     "type": "string",
-                    "required": true,
                     "description": "The programming language of the project. Either 'python' or 'cpp'"
                 },
                 {
                     "name": "repoType",
                     "type": "string",
-                    "required": true,
                     "description": "The type of the repository: 'app' or 'sdk'"
                 },
                 {
                     "name": "appManifestPath",
                     "type": "string",
-                    "required": true,
                     "description": "Path of the AppManifest file, relative to the .velocitas.json"
+                },
+                {
+                    "name": "githubRepoId",
+                    "type": "string",
+                    "description": "The id of the repository, e.g. myOrg/myRepo."
+                },
+                {
+                    "name": "appContainerPlatforms",
+                    "type": "string",
+                    "description": "Comma separated list of supported app container platforms.",
+                    "default": "linux/amd64, linux/arm64"
```

"appContainerPlatform" sound a bit being something like "Kubernetes" and such.
Maybe better call it "appExecutionPlatform" or "appExecutionArchitecture"
@BjoernAtBosch

```suggestion
                    "name": "appExecutionPlatforms",
                    "type": "string",
                    "description": "Comma separated list of supported app container platforms.",
                    "default": "linux/amd64, linux/arm64"
```